### DISCUSSION TITLE
Bug Report: Definition/References popup on Comment button text

### DISCUSSION THREAD

**Discussion by aarongable** [community/community#8677] [Pull Requests]
**Steps to reproduce**:
It is unclear if these steps will *reliably* reproduce the bug, but these are the steps that got me there:
1) Upload a PR which includes some deletions
2) Have another user comment on one of the deleted lines
3) In the "Files Changed" tab, begin drafting a reply to their comment
4) Mouse over the "Add Single Comment" button

**Expected result**:

5) The button changes color slightly, and can be clicked to add the comment.

**Actual result**:

5) The above, but **also** each individual word of the button gets highlighted in yellow when mousing over it.
6) Clicking directly on a word in the button text pulls up the Definitions/References popup, which should only apply to lines of code.

![image](https://user-images.githubusercontent.com/2474926/145623096-49d945f0-dde2-45f3-ae97-c77f69d95a19.png)

7) In addition, clicking the "Cancel" button both causes a malformed alert to appear (which would presumably normally contain the "are you sure you want to cancel your comment?" text), and then the Definitions/References popup when you click through the alert.

![image](https://user-images.githubusercontent.com/2474926/145623284-f65da2d9-c178-4057-89b7-9d20bcf41228.png)

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
